### ISSUE TITLE
NeuralNet(url: url) got nil

### ISSUE THREAD

**Issue by iThinkersTeam** [Swift-AI/NeuralNet-Handwriting-iOS#1]
Hello,

I tried to run app on the simulator. 
I got url but when I try get neuralNet = try NeuralNet(url: url) I got nil.

Error: 2017-06-17 09:34:58.582236+0300 Handwriting[5478:139399] fatal error: initialization("One or more required NeuralNet properties are missing."): file /Users/alex/Downloads/NeuralNet-Handwriting-iOS-master/Handwriting/MainViewController.swift, line 48

How can I solve it? 

Thanks

**Comment by D4ttatraya** [comment]
I am also getting same error. I am using Xcode9-beta with Swift3.2.
This is causing because [Error in NSNumber -> Float bridging in Swift3.2](https://bugs.swift.org/browse/SR-5179)

When I do this hard-coding (just for checking) in ```Storage.swift``` inside init function
```public convenience init(url: URL) throws```
**then it works fine**:
```
let layerNodeCounts = array[NeuralNet.layerNodeCountsKey] as! [Int]
let lr = (array[NeuralNet.learningRateKey] as! NSNumber).floatValue
let momentum = (array[NeuralNet.momentumKey] as! NSNumber).floatValue
let batchSize = array[NeuralNet.batchSizeKey] as! Int
let hiddenActivationStr = array[NeuralNet.hiddenActivationKey] as! String
let outputActivationStr = array[NeuralNet.outputActivationKey] as! String
let weights = (array[NeuralNet.weightsKey] as! [[NSNumber]]).map { $0.map { $0.floatValue } }
```

Somehow while reading data from ```neuralnet-mnist-trained``` file all ```Float``` are getting returned in ```NSNumber``` type.

**Comment by D4ttatraya** [comment]
Created [pull request#2](https://github.com/Swift-AI/NeuralNet-Handwriting-iOS/pull/2) by resolving issue.

**Comment by iThinkersTeam** [comment]
Thank you very much.

**Comment by D4ttatraya** [comment]
Let's wait till PR gets merged.

**Comment by collinhundley** [comment]
Merged #2
